clients/web/src/app.js

```diff
@@ -182,7 +182,6 @@ girder.App = girder.View.extend({
         girder.router.navigate(route, {trigger: true});
 
         if (girder.currentUser) {
-            girder.eventStream.close();
```

Comment:
I will add this back in to be safe, I figured this wouldn't ever be called if the event stream was already open, but that may happen in some corner cases.